src/status_im/contexts/wallet/collectible/tabs/about/view.cljs

```diff
@@ -25,12 +25,13 @@
       [quo/text
        {:size :paragraph-2}
        (:description collectible-data)]]
-     [quo/section-label
-      {:container-style style/section-label
-       :section         (i18n/label :t/on-the-web)}]
-     [rn/view {:style style/link-cards-container}
-      (for [item (:cards temp/collectible-about)]
-        ^{:key (:title item)}
-        [quo/link-card (assoc item :container-style link-card-container-style)])]]))
+     (when (count collectible-about)
+       [quo/section-label
+        {:container-style style/section-label
+         :section         (i18n/label :t/on-the-web)}]
+       [rn/view {:style style/link-cards-container}
+        (for [item (:cards collectible-about)]
+          ^{:key (:title item)}
```

Comment:
Not a blocker: I know this is not from your PR. I have a feeling that using the name of the collectable as the key is not a good idea as it may have the same names.

Reply:
Any suggestions?